index.html

```diff
@@ -68,6 +68,8 @@ <h1 class="noselect">sButtons</h1>
         <div class="subtitle">
           Simple buttons you can use easily for your next project.
         </div>
+        <a href="#" id="downloadGithubRawButton" class="sbtn click-btn btn-rounded">Download</a>
+        <a href="https://github.com/shahednasser/sbuttons" target="_blank" class="sbtn outline-btn btn-rounded black-btn">Documentation</a>
```

Comment:
I noticed you added a `black-btn` class here, but I don't think it's necessary.

Reply:
You got it! I'll make these changes :)